.github/workflows/script_test.yml

```diff
@@ -1,15 +1,13 @@
-name: Script Test
+name: Test install.sh
 on:
-  push:
-    branches:
-      - master
   pull_request:
     branches:
       - master
+    paths:
+      - 'install.sh'
```

Comment:
I am not sure do we need to test the `install.sh` not only on `install.sh` changes, but also like changes in Dockerfile and CRD.

PTAL @YangKeao @Hexilee @iguoyr

Reply:
Integration test will run `hack/local-up-chaos-mesh.sh`, and it calls `install.sh`, will it solve this problem?